### PR TITLE
Create verification route and verification functionality

### PR DESCRIPTION
**Pull request by carloslahrssen**
Created verification component and route so that users can now verify their account when given a token. 

This verify component parses the query string from the URL and then uses the verify mutation from the Graphql schema. Also changed the peach url to peach-hackfiu for the time being as I didn't have access to the peach alias on now.

### CODE REVIEW

File: src/components/Verify.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import { VERIFY } from '../graphql/mutations';
+import config from '../config';
+import qs from 'query-string';
+
+const EVENT_COLOR = config.EVENT_MAIN_COLOR;
+
+/* This class exists so that the mutation can take place when the component mounts
+   as per this issue: https://stackoverflow.com/questions/49456738/how-to-run-a-mutation-on-mount-with-react-apollo-2-1s-mutation-component
+*/
+class CallVerify extends React.Component {
```
Comment (david-castaneda) [inline src/components/Verify.js]: Is this extra class needed? Could you not add this piece of code in the Verify class component?

**Review by david-castaneda — COMMENTED** [review]
Noticed every " " changed with ' ' is this a linter setting you have enabled? Just wondering cause lots of files were marked as changed but the only thing changed was the double quotes to single, making it a bit more tedious to review tons of unchanged files.

File: src/components/Verify.js (hunk 1 above)
Comment (carloslahrssen) [reply src/components/Verify.js]: The reason I create another class is due to the fact that I need to perform the function once a component is mounted. Before attempting to use another Class component and just rendering through a function, there would be state changes occurring improperly throwing me an error. After searching through fixes, a common fix is to perform the function through another class. If you have any other ideas let me know.

**Comment by carloslahrssen** [comment]
Yeah, my prettier settings are causing it to fix, all these quotes. Would it be better for us to have .vscode settings folder for all of us to share the same styling?

**Comment by david-castaneda** [comment]
> Yeah, my prettier settings are causing it to fix, all these quotes. Would it be better for us to have .vscode settings folder for all of us to share the same styling?

Yea an issue should be opened up for a .vscode file

**Pull request merged**